Documentation/ceph-block.md

```diff
@@ -58,6 +58,8 @@ Create the storage class.
 kubectl create -f storageclass.yaml
 ```
 
+**NOTE** As [specified by Kubernetes](https://v1-9.docs.kubernetes.io/docs/concepts/storage/persistent-volumes/#retain), when using the `Retain` reclaim policy, the ceph RBD images that back up `PersistentVolume`s will continue to exist even after the PV is deleted, and have to be cleaned up manually using `rbd rm`.
```

Comment:
Perhaps https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy/ is a more stable link?

Reply:
So there are 3 entities involved here: PVC -> PV -> RBD image. From what I can gather, your link only specifies that when you delete a PVC with `Retain` reclaim policy, the PV is not deleted. The note I added is referring to the fact that if you also delete the PV, the RBD image is not deleted.

Reply:
@lbogdan Could you find a newer link? This is for 1.9, which is not supported by rook anymore.

Reply:
Changed to 1.13. The reasoning behind the 1.9 link was that was the first version describing `Retain` reclaim policy, and also that it didn't change since, and being an underlying concept, it most probably never will.